website/project/views/node.py

```diff
@@ -875,6 +876,7 @@ def serialize_collections(cgms, auth):
         'status': cgm.status,
         'type': cgm.collected_type,
         'is_public': cgm.collection.is_public,
+        'logo': cgm.collection.provider.asset_files.get(name='favicon-16x16').file.url
```

Comment:
TCI build's failing due to this line. I've got a helper method in #8431 that should fix. I'll make sure it gets scheduled for release no later than this PR, and update it here.

Also, judging from https://github.com/CenterForOpenScience/osf-assets/ , it seems like `-16x16` (and `32`) may have been deprecated in favor of just `favicon`.